app/include/zmk/hid.h

```diff
@@ -89,6 +90,23 @@ static const uint8_t zmk_hid_report_desc[] = {
     /* INPUT (Data,Ary,Abs) */
     HID_INPUT(0x00),
     HID_END_COLLECTION,
+
+    HID_USAGE_PAGE(HID_USAGE_GD),
+    HID_USAGE(HID_USAGE_GD_MOUSE),
+    HID_COLLECTION(HID_COLLECTION_APPLICATION),
+    HID_REPORT_ID(0x04),
+    HID_USAGE(HID_USAGE_GD_POINTER),
+    HID_COLLECTION(HID_COLLECTION_PHYSICAL),
+    HID_USAGE_PAGE(HID_USAGE_BUTTON),
+    HID_USAGE_MIN8(0x1),
+    HID_USAGE_MAX8(0x10),
```

Comment:
Is 0x10 hard coded here arbitrary, or based on some spec?

Reply:
It's sorta arbitrary. It's the number of buttons, and 8 was chosen to fill up a word. There's no real rhyme or reason, and i suspect most systems won't even support all 8 buttons.

Reply:
That's fine, can we pull out the number of supported buttons into a define value to name it?